source/common/umf_helpers.hpp

```diff
@@ -328,6 +351,28 @@ inline umf_result_t createMemoryProvider(
   return UMF_RESULT_SUCCESS;
 }
 
+inline umf_result_t createMemoryProxyPool(umf_memory_provider_handle_t provider,
```

Comment:
There is already a function to create proxy pool: https://github.com/oneapi-src/unified-runtime/blob/main/source/common/ur_pool_manager.hpp#L307